hazelcast/src/test/java/com/hazelcast/client/heartbeat/ClientHeartbeatTest.java

```diff
@@ -104,24 +104,19 @@ public void connectionRemoved(Connection connection) {
     }
 
     @Test
-    public void testInvocation_whenHeartbeatStopped() throws InterruptedException {
-        hazelcastFactory.newHazelcastInstance();
-        final HazelcastInstance client = hazelcastFactory.newHazelcastClient(getClientConfig());
-        final HazelcastInstance instance2 = hazelcastFactory.newHazelcastInstance();
```

Comment:
why did you change the logic of the test?

Reply:
We have changed the logic of the invocations. A put can be successfully done if there are any connected connection. This test was using two members and blocking only one.  I made it  one member it to make it throw `TargetDisconnectedException`